thetis/options.py

```diff
@@ -245,6 +245,12 @@ def __init__(self):
 
         Bottom stress is :math:`\tau_b/\rho_0 = -g \mu^2 |\mathbf{u}|\mathbf{u}/H^{1/3}`
         """
+        self.wd_alpha = None
+        r"""Coefficient or None: Wetting-drying parameter :math:`\alpha`
+
+        Used in bathymetry displacement function that ensures positive water depths. Unit is meters.
+        Default is None, which disables wetting and drying.
```

Comment:
I'd prefer setting a default value. Sanity checking of the options is difficult to maintain and it's not obvious where it should be done. You could mention in the documentation that the parameter is problem specific, and give guidelines on how to choose the value (if there is a rule).